### PR TITLE
Fix some issues on community pages

### PR DESCRIPTION
**Pull request by ginifizz**
| Q             | A
| ------------- | ---
| Bug fix?      | yes 
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

- fix image cache bug on random contributors
- fix contributor title when no name
- add core team badge
- fix accessibility headings
- expand header search bar on click on middle resolutions
- add readme for github features

image

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -214,8 +215,37 @@ const getAllMeetupEvents = async () => {
 const CONTRIBUTOR_NODE_TYPE = `Contributor`;
 const EVENT_NODE_TYPE = `Event`;
 
+const getOrganizationTeam = async (organizationName, teamName) => {
```
Comment (alanpoulain) [inline gatsby-node.js]:
```suggestion
const getOrganizationTeamMember = async (organizationName, teamName) => {
```

File: gatsby-node.js [hunk 2]
```diff
@@ -214,8 +215,37 @@ const getAllMeetupEvents = async () => {
...
+      data.map(async team => ({
+        ...team,
+        members: await getOrganizationTeam(organizationName, team.slug),
+      }))
+    );
+
+    return fullTeams;
+  } catch (error) {
+    console.error(
+      "UNAUTHORIZED: You have restricted rights to API Platform teams. You can't retrieve core teams members"
```
Comment (alanpoulain) [inline gatsby-node.js]:
```suggestion
      `UNAUTHORIZED: You have restricted rights to ${organizationName} teams. You can't retrieve teams members`
```

File: src/components/community/BigContributor.js [hunk 3]
```diff
@@ -12,11 +13,21 @@ const BigContributor = ({ contributor, size }) => (
...
+        <svg version="1.1" viewBox="0 0 520 520" preserveAspectRatio="xMinYMin meet">
+          <circle cx="260" cy="260" r="250" />
+        </svg>
+      </div>
+      {isCoreTeam(contributor) && (
+        <img
+          className="contributor__badge"
+          src="/badges/core-team.svg"
+          alt="core-team"
+          title="Membre de la core team"
```
Comment (alanpoulain) [inline src/components/community/BigContributor.js]:
```suggestion
          title="Core Team Member"
```

Comment (ginifizz) [reply src/components/community/BigContributor.js]: Oups !

File: src/styles/components/community/_contributors.scss [hunk 4]
```diff
@@ -159,6 +159,10 @@
     font-weight: $bold;
   }
 
+  .contributor__badge {
+    width:40px;
```
Comment (alanpoulain) [inline src/styles/components/community/_contributors.scss]:
```suggestion
    width: 40px;
```

We should add a CS fixer :stuck_out_tongue:

**Comment by alanpoulain** [comment]
Please do not merge yet, there is an issue with the GitHub token.

**Pull request merged**